rfcs/text/0008-threat-intel.md

```diff
@@ -23,30 +23,38 @@ Stage 1: Describe at a high level how this change affects fields. Which fieldset
 
 ### Proposed New Fields for Threat fieldset
 
-  * threat.time_first_seen
+  * threat.ioc.time_first_seen
     _The date and time when intelligence souce first reported sighting this indicator._
-  * threat.time_last_seen
+  * threat.ioc.time_last_seen
     _The date and time when intelligence source last reported sighting this indicator._
-  * threat.sightings
+  * threat.ioc.sightings
     _Number of times this indicator was observed conducting threat activity._
   * threat.type
     _Type of indicator as reprsented by Cyber Observable in STIX 2.0_
-  * threat.description
+  * threat.ioc.description
     _Describes the type of action conducted by the threat._
-  * threat.tlp
+  * threat.ioc.tlp
```

Comment:
In looking at what we're trying to accomplish, and discussing with @dcode @shimonmodi @dstepanic17 and looking at [marking in STIX 2.0](https://docs.oasis-open.org/cti/stix/v2.1/cs01/stix-v2.1-cs01.html#_95gfoglikdzh), we'd like to recommend `threat.marking.` This would allow for TLP and other classification markings like government classifications (SECRET, TOP SECRET, etc.), [PAP](https://www.misp-project.org/taxonomies.html#_pap), or other ISAC sharing/usage markings. 

This would also allow customers to add their own marking schemas.
```suggestion
threat.marking.tlp
```